### ISSUE TITLE
Round-tripping comments?

### ISSUE THREAD

**Issue by pchampin** [hjson/hjson-py#16]
According to the HJson FAQ:

> 
> Hjson allows you to round-trip your data, including your comments.
> 
> * But not yet with all implementations, PRs welcome :)

Unless I'm mistaken, `hjson-py` does not round-trip comments. Is there a plan to support it in the future?

I would gladly (try to) contribute a PR, so I had a look at the source. However it seems to me that such a feature would require a heavy refactoring of the data-structures involved, and possibly also of the parsing functions. Or did I miss something?

**Comment by dqsully** [comment]
No, I don't think hjson-py supports round-trip comments at the moment, and there isn't any active development on this feature. However, the [JavaScript](https://github.com/hjson/hjson-js) implementation does support round-trip comments.

**Comment by erwanp** [comment]
Hello ! I'd be interested for round-trip support to use hson as a configuration file in [an open-source physics code](https://github.com/radis/radis). 

Is this still not on the roadmap ? 
If not; do you know of any alternatives ? 

Best,

**Comment by sffc** [comment]
There's no roadmap for this, but a pull request would be appreciated!
